setup.py

```diff
@@ -21,6 +21,7 @@
     zip_safe=False,
     install_requires=['click', 'hubstorage', 'pip', 'requests', 'PyYAML',
                       'scrapinghub', 'six'],
+    extras_require={'docker-py': ['docker-py>=1.10.0']},
```

Comment:
It makes sense to keep using `shub` tool w/o using Docker at all, so `docker-py` should be an optional package here, I'll return a check for docker-py import to disable some click commands.